server/apps/expand.go

```diff
@@ -101,7 +103,23 @@ func (e *expander) collectConfig(expand *Expand) error {
 	return nil
 }
 
-func (e *expander) collectChannel(channelID string) func(*Expand) error {
+func (e *expander) collectTeam(teamID string) func(*Expand) error {
```

Comment:
@levb I added this function back into the expander because leaving/joining a team, does not have a "channel".  

However, I left the `collectChannelAndTeam` function because, given a channel, you can retrieve the team.